src/main/java/core/basesyntax/service/strategy/handlers/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.strategy.handlers;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    @Override
+    public void doOperation(FruitTransaction fruitTransaction) {
+        Storage.addToDataBase(fruitTransaction.getFruit(),
+                Storage.getByKey(fruitTransaction.getFruit()) - fruitTransaction.getQuantity());
```

Comment:
in situation where we don't have enough fruits to purchase we need to throw an exception
let's create custom exception for this purpose and throw it here